src/media.ts

```diff
@@ -0,0 +1,8 @@
+export const devices = {
```

Comment:
Should @oyabun confirm these breakpoints?

Reply:
He did.